### PR TITLE
Updating readme to reflect new bastion login workflow

### PR DESCRIPTION
**Pull request by endakelly**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -88,14 +88,11 @@ More process details to follow, it's currently being worked out.
...
 
-```shell
-Host *.platform.hmcts.net
-  User <your-email>@hmcts.net # must be lower case
-  PubkeyAuthentication no
-  ForwardAgent yes
-```
+Ensure ssh extension for the Azure CLI is installed: 'az extension add --name ssh'
+
+Run 'az ssh config --ip \* --file ~/.ssh/config'
```
Comment (timja) [inline README.md:95]: we should not be modifying all values, we've already had people hit issues because of this

```suggestion
Run `az ssh config --ip \*.platform.hmcts.net --file ~/.ssh/config`
```

File: README.md [hunk 2]
```diff
@@ -139,7 +136,7 @@ _Note: it's also possible to tunnel the connection to your own machine and use o
 # you can get this from the portal, or determine it via the inputs your pass to this module in your code
 POSTGRES_HOST=rpe-draft-store-aat.postgres.database.azure.com
 
-ssh -N bastion-dev-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
+ssh bastion-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
```
Comment (timja) [inline README.md:139]: why did you remove this?
```suggestion
ssh -N bastion-nonprod.platform.hmcts.net -L 5440:${POSTGRES_HOST}:5432
```

File: README.md [hunk 3]
```diff
@@ -168,16 +165,11 @@ psql "sslmode=require host=localhost port=5440 dbname=${DB_NAME} user=${DB_USER}
...
-Update the user line to your email address
+Ensure ssh extension for the Azure CLI is installed: 'az extension add --name ssh'
 
-```shell
-Host *.platform.hmcts.net
-  User <your-email>@hmcts.net # must be lower case
-  PubkeyAuthentication no
-  ForwardAgent yes
-```
+Run 'az ssh config --ip \* --file ~/.ssh/config'
```
Comment (timja) [inline README.md:172]:
```suggestion
Run `az ssh config --ip \*.platform.hmcts.net --file ~/.ssh/config`
```

File: README.md [hunk 4]
```diff
@@ -168,16 +165,11 @@ psql "sslmode=require host=localhost port=5440 dbname=${DB_NAME} user=${DB_USER}
 
 <details>
 
-<summary>Bastion configuration</summary>
+Run 'az login'
```
Comment (timja) [inline README.md:168]: isn't there a minimum version for azure cli?

File: README.md (hunk 2 above)
Comment (endakelly) [reply README.md:139]: -N switch doesn't seem to work for me so figured it wasn't necessary - could be just my ssh playing up

File: README.md (hunk 4 above)
Comment (endakelly) [reply README.md:172]: Updated

File: README.md [hunk 5]
```diff
@@ -88,14 +88,13 @@ More process details to follow, it's currently being worked out.
 
 <summary>Bastion configuration</summary>
 
-Update the user line to your email address
+Ensure you have Azure CLI version 2.22.1 or later installed
```
Comment (timja) [inline README.md:91]: this line needs removing 'Add SSH config, this goes in `~/.ssh/config`, create the file if it doesn't exist'

File: README.md [hunk 6]
```diff
@@ -88,14 +88,13 @@ More process details to follow, it's currently being worked out.
...
 
-```shell
-Host *.platform.hmcts.net
-  User <your-email>@hmcts.net # must be lower case
-  PubkeyAuthentication no
-  ForwardAgent yes
-```
+Run 'az login'
```
Comment (timja) [inline README.md:93]:
```suggestion
Run `az login`
```

File: README.md (hunk 4 above)
Comment (timja) [reply README.md:172]:
```suggestion
Run `az login`
```

**Review by timja — APPROVED** [review]
